src/components/Tabs/Tabs.module.css

```diff
@@ -2,3 +2,21 @@
   top: 0;
   z-index: 100;
 }
+
+.page-section-tabs :global(.pf-c-tabs) {
+  min-height: 42px;
+}
+
+.page-section-tabs :global(.pf-c-label) {
```

Comment:
I can see you decided to implement `TabsLabel` component I proposed, in this case you should move also part of the related stylesheets (nested ones) into new file `TabsLabel.module.css`:

```css
.tabs-label :global(.pf-c-label) { /* ... */ }

/* etc */
```